poucave/utils.py

```diff
@@ -363,21 +365,19 @@ def on(self, event, callback):
         self.callbacks.setdefault(event, []).append(callback)
 
 
-_bqclient = None
-
-
 async def fetch_bigquery(sql):  # pragma: nocover
     """
     Execute specified SQL and return rows.
     """
+    bqclient = getattr(threadlocal, "bqclient", None)
+    if bqclient is None:
+        # Reads credentials from env and connects.
+        bqclient = bigquery.Client()
+        setattr(threadlocal, "bqclient", bqclient)
```

Comment:
Nit: I'd usually write this as
```suggestion
        threadlocal.bqclient = bqclient
```

Reply:
Hmm, I like to highlight that we set a custom attribute here